test/test_netcdf_writer.jl

```diff
@@ -2673,6 +2675,50 @@ function test_netcdf_free_surface_mixed_output(arch)
     return nothing
 end
 
+function test_netcdf_buoyancy_force(arch)
+
+    Nx, Nz = 8, 8
+    Hx, Hz = 2, 3
+    Lx, H  = 2, 1
+
+    grid = RectilinearGrid(arch,
+                           topology = (Periodic, Flat, Bounded),
+                           size = (Nx, Nz),
+                           halo = (Hx, Hz),
+                           x = (-Lx, Lx),
+                           z = (-H, 0))
+
+    Boussinesq_eos = (TEOS10EquationOfState(),
+                        RoquetEquationOfState(:Linear),
+                        RoquetEquationOfState(:Cabbeling),
+                        RoquetEquationOfState(:CabbelingThermobaricity),
+                        RoquetEquationOfState(:SecondOrder),
+                        RoquetEquationOfState(:SimplestRealistic))
```

Comment:
```suggestion
    Boussinesq_eos = (TEOS10EquationOfState(),
                      RoquetEquationOfState(:Linear),
                      RoquetEquationOfState(:Cabbeling),
                      RoquetEquationOfState(:CabbelingThermobaricity),
                      RoquetEquationOfState(:SecondOrder),
                      RoquetEquationOfState(:SimplestRealistic))
```

Reply:
I don't know that much about different options for equations of state, but does this cover all cases that are currently available? I know this is implicitly tested elsewhere but shouldn't we add a linear EOS there for good measure/completeness?

Reply:
I forgot the the `:Freezing` coefficient set sorry! After that all the options from SeawaterPolynomials.jl are included. Did you mean to add `LinearEquationOfState` from the Oceananigans package?